src/Snap/Internal/Http/Server/Config.hs

```diff
@@ -130,6 +138,15 @@ instance Show ConfigLog where
     show (ConfigFileLog f) = "log to file " ++ show f
     show (ConfigIoLog _)   = "custom logging handler"
 
+------------------------------------------------------------------------------
+-- | This handler may be used (in conjunction with setErrorLogHandler) to write out error logs in a
+-- custom manner.
+type ErrorLogHandler = ByteString -> IO ByteString
```

Comment:
Wow, it deleted my previous changes here when I updated the PR. Not used to that. I do most of my work on BitBucket. So, what did you mean specifically about this type? Did you want me to:

 1. Move it to the Types.hs module OR
 1. Rename the type to 'data ErrorLogHandler = ErrorLogHandler (ByteString -> IO ByteString)
 1. Delete it altogether and use (ByteString -> IO ByteString) everywhere?

I am just not sure what your previous comment here was asking for. Cheers.